packages/rmf-models/generate-models.sh

```diff
@@ -1,11 +1,15 @@
 #!/bin/bash
```

Comment:
Yeah, although this is up for debate if we should commit the model files or generate it on the fly at bootstrap/install. Committing the files makes it easier for other packages but makes it harder if you want to develop both rmf and rmf-web together, because you can't generate new models without the rmf changes merged first.